src/main/java/io/fabric8/jenkins/openshiftsync/JenkinsUtils.java

```diff
@@ -119,17 +120,42 @@ public static String getRootUrl() {
 		return root;
 	}
 
-	public static boolean verifyEnvVars(Map<String, ParameterDefinition> paramMap, WorkflowJob workflowJob) {
+	public static boolean verifyEnvVars(Map<String, ParameterDefinition> paramMap, WorkflowJob workflowJob) throws AbortException {
         if (paramMap != null) {
             String fullName = workflowJob.getFullName();
-            WorkflowJob job = Jenkins.getActiveInstance()
-                    .getItemByFullName(fullName,
-                            WorkflowJob.class);
-            if (job == null) {
+            WorkflowJob job = null;
+            long now = System.currentTimeMillis();
+            boolean anyErrors = false;
+            while (System.currentTimeMillis() - 5000 > now) {
```

Comment:
refactor: _may improve readbility_
it's better to give semantic meaning for it?
```
while(forSeconds(5)) {
....
}

boolean forSeconds(int seconds) {
  return System.currentTimeMillis() - seconds > now
}
```